torch/csrc/distributed/rpc/rref_context.cpp

```diff
@@ -76,12 +76,11 @@ std::shared_ptr<UserRRef<T>> RRefContext::createUserRRef(
   // RRefContext does not track user RRefs, it will be destructed when there
   // is no shared_ptrs pointing to it. NB: cannot use make_shared here as the
   // constructor of UserRRef is private
-  auto userRRef =
+  return
       std::shared_ptr<UserRRef<T>>(new UserRRef<T>(ownerId, rrefId, forkId));
-  if (forkId.createdOn_ != ownerId) {
-    addPendingUser(forkId, userRRef);
-  }
-  return userRRef;
+  //if (forkId.createdOn_ != ownerId) {
```

Comment:
Why did this get commented out? If this is really the intention i think you should add a comment explicitly on why these lines are commented out.

Do I understand correctly that "pending_user" should always be used when you send a fork from a user to anyone else? IN this case we should addPendingUser, right?

Reply:
My bad. This should be removed. This was added when I was trying address @gqchen's comments and debug:

> this assumes the created RRef will be send over RPC. I'm not sure we want to assume this in this portion of the code. Should we consider moving this logic out of this function to the caller of this function where we indeed is going to make RPC calls.

This is now moved to `python_functions.cpp` where the RPC call occurs.